cmd/operator/deploy/operator/12-alertmanager.yaml

```diff
@@ -82,8 +82,8 @@ spec:
           limits:
             memory: 128M
           requests:
-            cpu: 8m
-            memory: 32M
+            cpu: 1m
+            memory: 16M
```

Comment:
Should we maybe keep this consistent with `examples/alertmanager.yaml`?

Reply:
Thx - done!